docs/openapi.md

```diff
@@ -0,0 +1,82 @@
+## Import API definition to 3scale from OpenAPI definition
+
+Features:
+
+* OpenAPI __2.0__ specification (f.k.a. __swagger__)
+* Create a new service. New service name will be taken from openapi definition `info.title` field.
+* Update existing service, providing `SERVICE_ID` with `--service` option.
+* Create methods in the 'Definition' section. Method names are taken from `operation.operationId` field.
+* Attach newly created methods to the *Hits* metric.
+* Create mapping rules and show them under `API > Integration`.
```

Comment:
Is it going to keep the `/ => hits` mapping?

Reply:
Yeah, target service mapping rules are not deleted. Both when new service is created or existing service is updated.

It is behaving with semantics of HTTP `POST`. Should be HTTP `PUT` semantics?

Reply:
I'm not sure if this is about PUT or POST. 

Having the default mapping rule `GET / => hits` will match every GET request and also the mapping rule for the operation ID and reporting both values to backend resulting in double reporting.

Reply:
Will remove existing mapping_rules as default behavior. Maybe with a flag to avoid removing when set.